lib/cli.js

```diff
@@ -40,11 +40,22 @@ function parse(path) {
 }
 
 function processFile(filePath) {
-  let source = fs.readFileSync(filePath, "utf8");
-  let formatted = formatter.formatJson(source);
-  let fileWithErrors = showDiff(program.quiet, source, formatted, filePath);
+  let formatted, source, fileWithErrors;
 
-  errors.push(fileWithErrors);
+  try {
+    source = fs.readFileSync(filePath, "utf8");
+    formatted = formatter.formatJson(source);
+    fileWithErrors = showDiff(program.quiet, source, formatted, filePath);
+
+    errors.push(fileWithErrors);
+  } catch (e) {
+    console.log("Couldn't read the file / directory");
+    console.log("Try to:");
+    console.log("  - use quotes (eg: jsoncs 'my/file.json')");
+    console.log("  - use glob pattern (eg: jsoncs 'my/*.json')");
+    console.log("  - use glob pattern (eg: jsoncs 'my/*/*.json')");
+    process.exit(1);
+  }
 
   if (program.fix) {
```

Comment:
This could go to inside the `try` block. And maybe extract it to it's own function.